serverlessworkflow/modules/ROOT/pages/cloud/build-workflow-image-with-quarkus-cli.adoc

```diff
@@ -1 +1,173 @@
-= Building Workflow Images locally with Quarkus CLI
\ No newline at end of file
+Building Workflow Images locally with Quarkus CLI
+=================================================
+Author Name <fspolti@redhat.com>
+v1.0, 06-03-2022
+:compat-mode!:
+// Metadata:
+:description: Build Serverless Application with Quarkus CLI
+:keywords: kogito, workflow, quarkus, serverless, quarkus-cli
+// links
+:quarkus-kogito-examples: https://github.com/kiegroup/kogito-examples/tree/stable/kogito-quarkus-examples
+:example-project: https://github.com/kiegroup/kogito-examples/tree/stable/kogito-quarkus-examples/serverless-workflow-greeting-quarkus
+:quarkus-cli: https://quarkus.io/guides/cli-tooling
+:quarkus-container-images: https://quarkus.io/guides/container-image
+
+
+On this guild you will learn how to build a Serverless Application using *Quarkus CLI*.
+
+== Pre Requisites
+
+Before starting make sure you have the needed dependencies installed.
+For this tutorial we assume that you already have your Serverless Workflow ready to be built inside
+a container image. If not, you can use some of the {quarkus-kogito-examples}[examples], for this tutorial we will be using the {example-project}[serverless-workflow-greeting-quarkus].
+
+What will you need:
+
+- Docker - recommended to use always the latest version.
+    * if you don't have it installed or can't install it, don't worry, `JiB` allows you to build Container images wihtouth having the client tooling to build containers, however, it will require the push the built image to a registry, which is described in the following lines.
+- {quarkus-cli}[Quarkus CLI] {quarkus_version}
+- Java {java_min_version}
+- Maven {maven_min_version}
+- GraalVM {graalvm_min_version}: Note that, GraalVM is option if you already have the expected version installed locally.
+
+
+== Getting started
+
+First, let's clone the {kogito-examples}[kogito-examples] repository and access the {example-project}[serverless-workflow-greeting-quarkus] example.
+
+.Cloning Kogito Examples
+[source,shell]
+----
+$ git clone --branch stable {kogito-examples} && kogito-examples/kogito-quarkus-examples/serverless-workflow-greeting-quarkus
+----
+
+If you wish to run the example beforehand, follow the steps described on the example.
+
+With the example in place, let's visit {quarkus-cli}[Quarkus CLI], follow the installation instructions based on your preferred tool. Make sure you have the following Quarkus version installed:
```

Comment:
I think this procedure will be covered here: https://github.com/kiegroup/kogito-docs/blob/main/serverlessworkflow/modules/ROOT/pages/getting-started/getting-familiar-with-our-tooling.adoc

So we can skip this explanation now, and add a link to that other guide.

"Assumed that you already have the Quarkus tooling installed, if not please visit: ..."

Reply:
+1